src/cryptoadvance/specter/managers/node_manager.py

```diff
@@ -137,6 +137,8 @@ def add_node(
     def add_internal_node(
         self,
         name,
+        network="mainnet",
```

Comment:
Why not `"main"`? Looks like we have now main and mainnet that are basically the same.

Reply:
Changed